chart_review/cohort.py

```diff
@@ -69,7 +69,7 @@ def _collect_note_ranges(
                     ls_id = ignore_id  # must be direct note ID
                 else:
                     # Must just be over-zealous excluding (like automatically from SQL)
-                    continue
+                    continue  # pragma: no cover
```

Comment:
There's some issue with python 3.9 that this line doesn't get hit on, but other pythons do. Rather than debug this ancient version (which is eol in a couple months), just pragma this and move on - it is tested.